connectors/kafka/pom.xml

```diff
@@ -31,16 +31,30 @@ except in compliance with the proprietary license.</license.inlineheader>
   </properties>
 
   <dependencies>
+    <dependency>
+      <groupId>com.google.code.gson</groupId>
+      <artifactId>gson</artifactId>
+    </dependency>
+
+    <dependency>
+      <groupId>com.fasterxml.jackson.core</groupId>
+      <artifactId>jackson-annotations</artifactId>
+    </dependency>
```

Comment:
Do we need `gson` and `jackson-annotation`?

Reply:
Yes, we need because we need jackson annotation to indicate Zeebe how to resolve the enum (Zeebe uses jackson for deserialization). We should remove gson in the future probably.